dace/frontend/python/newast.py

```diff
@@ -3589,6 +2759,10 @@ def visit_Call(self, node: ast.Call):
 
         result = func(self.sdfg, self.last_state, *args, **keywords)
 
+        if isinstance(result, tuple) and type(result[0]) is nested_call.NestedCall:
```

Comment:
why not `isinstance(result[0], nested_call.NestedCall)`?

Reply:
That would break things like reduce, which return `[]`, because `result[0]` would give an index error.